setup.py

```diff
@@ -36,9 +38,11 @@
 
 EXTRAS_REQUIRE = {
     'doc': [
-        'sphinx==1.8.5',
-        'sphinx_rtd_theme>=0.5.0',
+        'sphinx==3.2',
+        'jinja2<3.1',
+        'sphinx_rtd_theme==0.5.2',
         'ipython',
+        'sphinx-gallery==0.8.1'
```

Comment:
`sphinx-gallery` doesn't seem necessary (yet?)

Reply:
Removed dependency in commit https://github.com/NREL/Panel-Segmentation/pull/15/commits/41ecea125c343fc3db41b057994ba949b468f77b